docker-compose-avro.yml

```diff
@@ -44,10 +44,10 @@ services:
       - --addr=0.0.0.0:8300
       - --pd=http://upstream-pd:2379
       - --log-file=/logs/capturer0.log
-      - --log-level=debug
+      - --log-level=info
```

Comment:
should we keep debug log level for ticdc

Reply:
yes, this is helpful.